src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -209,109 +210,118 @@ const SubscriberHistory = ({
+                      summary={_('subscriber_history.table_result.aria_label_table')}
+                    >
+                      <thead>
+                        <tr>
+                          <th scope="col">
+                            <FormattedMessage id="master_subscriber_sent_campaigns.grid_campaign" />
+                          </th>
+                          <th scope="col">
+                            <FormattedMessage id="master_subscriber_sent_campaigns.grid_subject" />
+                          </th>
+                          <th scope="col">
+                            <FormattedMessage id="master_subscriber_sent_campaigns.grid_delivery" />
+                          </th>
+                          <th scope="col">
+                            <FormattedMessage id="master_subscriber_sent_campaigns.grid_clicks" />
+                          </th>
+                        </tr>
+                      </thead>
+                      <tfoot>
+                        <tr>
+                          <td colSpan="4" style={{ textAlign: 'right' }}>
+                            <Pagination
+                              currentPage={state.currentPage}
+                              pagesCount={state.pagesCount}
+                              urlToGo={`/reports/subscriber-history?email=${state.subscriber.email}&`}
+                            />
+                          </td>
+                        </tr>
+                      </tfoot>
+                      <tbody>
+                        {state.sentCampaigns.length ? (
+                          <>
+                            {state.sentCampaigns.map((campaign, index) => (
+                              <tr key={index}>
+                                <td>
+                                  {campaign.urlImgPreview ? (
+                                    <div className="dp-tooltip-container">
+                                      <a
+                                        href={`${reportsUrl}/Dashboard.aspx?idCampaign=${campaign.campaignId}`}
+                                      >
+                                        {campaign.campaignName}
+                                        <div className="dp-tooltip-block">
+                                          <img
+                                            src={campaign.urlImgPreview}
+                                            alt={_('subscriber_history.alt_image')}
+                                          />
+                                        </div>
+                                      </a>
+                                    </div>
+                                  ) : (
                                     <a
-                                      href={`${reportsUrl}/Dashboard.aspx?idCampaign=${campaign.campaignId}`}
+                                      href={`https://reports2.fromdoppler.com/Dashboard.aspx?idCampaign=${campaign.campaignId}`}
```

Comment:
🤔

Reply:
I have to make a rebase 😅

Reply:
Or not... I don't know what happens here 😕

Reply:
I didn't change anything there. Github mark this as a change but it's not.

Reply:
Mmmm, something happens here

Reply:
I've just merged it 😢
This is something we have to change surely but I can do it in other PR with only refactors (without any visual changes0